### PR TITLE
Add '.herokuapp.com' to ALLOWED_HOSTS

### PR DESCRIPTION
**Pull request by RyanBalfanz**

### CODE REVIEW

File: main/settings_prod.py [hunk 1]
```diff
@@ -34,7 +34,11 @@
 
 DATABASES['default']['CONN_MAX_AGE'] = 3600
 
-ALLOWED_HOSTS = ['.dicpick.herokuapp.com', '.dicpick.com']
+ALLOWED_HOSTS = [
+    '.herokuapp.com',
```
Comment (benjyw) [inline main/settings_prod.py]: Why is this necessary? Seems overly permissive, no?

**Comment by RyanBalfanz** [comment]
Until the settings are more usable and heroku is fully configured this is
the only way to get Review Apps working on heroku (apart from adding the
FQDN each new PR, of course, but that's bananas).

**Comment by benjyw** [comment]
Gotcha.
